k9mail/src/main/res/values/strings.xml

```diff
@@ -1192,36 +1192,46 @@ Please submit bug reports, contribute new features and ask questions at
 
-    <string name="crypto_msg_disabled">Message is not encrypted</string>
-
-    <string name="crypto_msg_encrypted_error">Message is encrypted, but there was a decryption error.</string>
-    <string name="crypto_msg_encrypted_insecure">Message is encrypted but insecure!</string>
-    <string name="crypto_msg_signed_error">Message is signed, but the signature could not be verified.</string>
-    <string name="crypto_msg_encrypted_no_provider">Message is encrypted, but no crypto app is configured.</string>
-    <string name="crypto_msg_unsupported_encrypted">Message is encrypted, but in an unsupported format.</string>
-    <string name="crypto_msg_unsupported_signed">Message is signed, but in an unsupported format.</string>
-    <string name="crypto_msg_incomplete_encrypted">Message is encrypted, but must be fully downloaded for decryption.</string>
-    <string name="crypto_msg_cancelled">Message is encrypted, but decryption was cancelled.</string>
-
-    <string name="crypto_msg_signed_unencrypted">Message is signed, but not encrypted.</string>
-    <string name="crypto_msg_encrypted_unsigned">Message was encrypted, but not signed.</string>
-
-    <string name="crypto_msg_sign_unknown">Signature made by unknown key.</string>
-    <string name="crypto_msg_sign_verified">Signature made by trusted key.</string>
-    <string name="crypto_msg_sign_unverified">Signature made by untrusted key!</string>
-    <string name="crypto_msg_sign_error">Signature contained an error.</string>
-    <string name="crypto_msg_sign_mismatch">Signature made by wrong key.</string>
-    <string name="crypto_msg_sign_expired">Signature made by expired key.</string>
-    <string name="crypto_msg_sign_revoked">Signature made by revoked key.</string>
-    <string name="crypto_msg_sign_insecure">Signature made by insecure key.</string>
-    <string name="crypto_msg_sign_incomplete">Message must be fully downloaded to verify its signature.</string>
-
-    <string name="crypto_msg_signed_encrypted">Message is signed and encrypted.</string>
-    <string name="crypto_msg_unsigned_encrypted">There is no signature, this message may have been intercepted.</string>
-
+    <string name="crypto_msg_title_plaintext">Plaintext</string>
+    <string name="crypto_msg_unencrypted_sign_error">end-to-end signature contained an error</string>
+    <string name="crypto_msg_incomplete_signed">must fully download message to process signature</string>
+    <string name="crypto_msg_unsupported_signed">contains unsupported end-to-end signature</string>
+
+    <string name="crypto_msg_title_unencrypted_signed_e2e">End-to-End signed plaintext</string>
+    <string name="crypto_msg_unencrypted_sign_verified">from verified signer</string>
+
+    <string name="crypto_msg_title_unencrypted_signed">Signed plaintext</string>
+    <string name="crypto_msg_unencrypted_sign_mismatch">but end-to-end key didn\'t match sender</string>
```

Comment:
From a translation point of view it's probably not a good idea to split up the error text like this. There's a good chance this can't be done in a similar way in at least one of the supported languages.

Reply:
The intent of splitting a general status (encrypted / e2e encrypted / signed plaintext / plaintext) should be transferable, at least I hope so. Other languages don't need to do the same partial sentence thing, if it works better for them.

I guess we will see from Transifex issues.